openinfra_core/pom.xml

```diff
@@ -108,6 +108,14 @@
 		    <artifactId>jersey-media-json-jackson</artifactId>
 		</dependency>
 		
+		<!-- simple json string to object converter -->

```

Comment:
We already implement Jackson as JSON library for Java. I don't understand why we should implement a second framework.